cipd_packages/ruby/third_party/ruby_ship/ruby_ship_build.sh

```diff
@@ -1,243 +1,248 @@
 #!/usr/bin/env bash
-#VERIFY PARAMETERS
-if [ $# -eq 0 ]
-  then
+
+set -e
+set -x
+
+function remove_dylib_signatures() {
+  # Remove signatures
+  ls $DIR/../build/bin/darwin_ruby/dylibs/* | xargs codesign --remove-signature;
+  # Resign with adhoc
+  ls $DIR/../build/bin/darwin_ruby/dylibs/* | xargs codesign --force -s -;
+}
```

Comment:
I hate using function parameters in bash, but I still feel like if/when we change this path we'll forget to update this function, and someone's day will be wasted debugging:

```suggestion
function remove_dylib_signatures() {
  if [[ -z "$1" ]]; then
    echo 'remove_dylib_signatures must be called with the path to the dylibs dir as an argument' >&2
    exit 42
  fi
  
  # Remove signatures
  ls "$1/*" | xargs codesign --remove-signature;
  # Resign with adhoc
  ls "$1/*" | xargs codesign --force -s -;
}
```

Reply:
And then updating calls to pass the path

Reply:
Updated!